src/test-edit.js

```diff
@@ -0,0 +1,39 @@
+// import { expect, jest, test } from '@jest/globals';
```

Comment:
Great job team 🤝

- [ ] [Optional] I would also recommend you remove this file from your project since it is not being used.